### PR TITLE
Xero Accounting

### PR DESCRIPTION
**Pull request by troystaylor**
---
When submitting a connector, please check the following conditions for your PR.

- [X] I attest that the connector works and I verified by deploying and testing all the operations.
- [X] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [X] I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`.

### CODE REVIEW

File: independent-publisher-connectors/Xero Accounting/Getting Started with Xero App Webhooks.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Getting Started with Xero App webhooks
```
Comment (sriyen-msft) [inline independent-publisher-connectors/Xero Accounting/Getting Started with Xero App Webhooks.md:1]: @troystaylor Please include this file content into `readme.md` file.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

**Comment by troystaylor** [comment]
@sriyen-msft Fixed

**Review by sriyen-msft — APPROVED** [review]

**Review by sriyen-msft — CHANGES_REQUESTED** [review]
Deployment Pending

**Review by sriyen-msft — COMMENTED** [review]
Deployment Pending

**Comment by mamurshe** [comment]
[[certify-connector]]

**Comment by mamurshe** [comment]
[[certify-connector]]

**Comment by mamurshe** [comment]
[[certify-connector]]

**Comment by mamurshe** [comment]
[[certify-connector]]

**Comment by troystaylor** [comment]
@natalie-pienkowska With your permission, I would like to close this and the other two Xero PRs. I would like to completely re-write these connectors using the current 2.x version Xero-provided OpenAPI definitions: https://github.com/XeroAPI/Xero-OpenAPI

**Comment by natalie-pienkowska** [comment]
Hi Troy! You're good to go :) Thanks so much for making sure our connectors are up to date and of the highest quality!

**Pull request closed without merging**